.pre-commit-config.yaml

```diff
@@ -41,6 +41,7 @@ repos:
     rev: 0.17.1
     hooks:
     - id: gersemi
+      args: ["-c"]
```

Comment:
This ensures that gersemi run in check mode which returns 1 if any formatting required and 0 otherwise